src/main/java/thePackmaster/patches/OpeningRunScreenPatch.java

```diff
@@ -15,8 +17,10 @@ public class OpeningRunScreenPatch {
             locator = Locator.class
     )
     public static void SetTheThing(NeowEvent __instance) {
-        BaseMod.logger.info("Packmaster is Ready To Do Thing");
-        SpireAnniversary5Mod.readyToDoThing = true;
+        if (AbstractDungeon.player.chosenClass == ThePackmaster.Enums.THE_PACKMASTER) {
```

Comment:
This was an unrelated issue I fixed along the way -- the packmaster logic was being run for all characters, so if you tried to start a run with a non-packmaster character it would crash.